src/Pororoca.Desktop/ViewModels/WebSocketConnectionViewModel.cs

```diff
@@ -734,5 +746,28 @@ static string GenerateDefaultInitialFileName(string wsName, int index, WebSocket
         }
     }
 
+    private async Task SaveAllExchangedMessagesToFilesAsync()
+        {
+            if (!ExchangedMessages.Any())
+                return;
+
+            string? destinationFolderPath = await FileExporterImporter.SelectFolderAsync();
+            if (destinationFolderPath is null)
+                return;
+
+            for (int i = 0; i < ExchangedMessages.Count; i++)
+            {
+                var vm = ExchangedMessages[ExchangedMessages.Count - 1 - i];
```

Comment:
Why the index selection here is `ExchangedMessages.Count - 1 - i`?
Can't it be just `i`?